lib/wrappers/transform-node.js

```diff
@@ -19,6 +19,11 @@ module.exports = class TransformNodeWrapper extends NodeWrapper {
     // If the any inputNode's ref does not match what is stored in here then we
     // know a modification has happened so we call the build method
     this.inputRevisions = new WeakMap();
+    this.nodeMap = new WeakMap();
```

Comment:
This could be rolled into `this.inputRevisions` if we don't want to create another weakmap.

Reply:
Fixed